.github/workflows/main.yml

```diff
@@ -0,0 +1,247 @@
+name: Main
+
+on:
+  push:
+  pull_request:
```

Comment:
Is this on purpose? As it is, it is running CI on any `push` and `pull_request`.

Reply:
Ah darn, no it's not, my mistake, fixed with [920ed9f](https://github.com/solana-program/single-pool/pull/2/commits/920ed9f8718e77322ff2e900dc196635d3625129)